app/model/DigitalEdition.scala

```diff
@@ -0,0 +1,25 @@
+package model
+
+import configuration.Config
+
+case class DigitalEdition(id: String, name: String, price: String, campaign: String)
+
+
+object DigitalEdition {
+
+  trait UrlConfig {
+    val externalSubscriptionUrl: String
+  }
+
+  object UK extends DigitalEdition("uk", "UK", "£11.99", "dis_2408")
+  object US extends DigitalEdition("us", "US", "$19.99", "dis_2378")
+  object AU extends DigitalEdition("au", "Australia", "$21.50", "dis_2379")
+  implicit lazy val defaultUrlConfig: UrlConfig = Config
+
+  def getRedirect(edition: DigitalEdition)(implicit urlConfig: UrlConfig): String = {
```

Comment:
ah thats interesting I don't have access to that repo!
And yes, I think you might be right about the config on second thoughts, I think I'll revert to 58afade, Joe and I just had a brief chat about it